docs/how-to/code/create-slice/openssl.yaml

```diff
@@ -2,10 +2,10 @@ package: openssl
 slices:
     bins:
         essential:
```

Comment:
how come it fixed the `essential` list but not the `contents` dict?

Reply:
well yes, if we're going to lint this we should be consistent right? Otherwise I don't see the point